src/main.cpp

```diff
@@ -1807,17 +1807,21 @@ void askForInternetPermission()
         }   // onCancel
     };   // ConfirmServer
 
-    GUIEngine::ModalDialog *dialog =
+    MessageDialog *dialog =
     new MessageDialog(_("SuperTuxKart may connect to a server "
         "to download add-ons and notify you of updates. We also collect "
         "anonymous hardware statistics to help with the development of STK. "
         "Please read our privacy policy at http://privacy.supertuxkart.net. "
         "Would you like this feature to be enabled? (To change this setting "
         "at a later time, go to options, select tab "
-        "'User Interface', and edit \"Connect to the "
+        "'General', and edit \"Connect to the "
         "Internet\" and \"Send anonymous HW statistics\")."),
         MessageDialog::MESSAGE_DIALOG_YESNO,
         new ConfirmServer(), true, true, 0.7f, 0.7f);
+
+    // Changes the default focus to be 'cancel', which is not
```

Comment:
You meant that the current "ok" default focus is not compliant ?

Reply:
I believe so - gdpr  requires a deliberate choice by the user to opt-in. With 'ok' being the default, and assuming that just pressing enter is not a deliberate choice (given that opt-out requires a click and pressing enter). IANAL, and if anyone can point out a counter example I am happy to leave the default, I don't like this.
We could add e.g. internet-permission as part of the installation process, but that's a lot of work and only covers a part of our user base :(
TBH, if we agree that we don't need the HW stats (has anyone looked at them in the last two years??), taking this out would be a really good choice imho.

Reply:
@leyyin had disabled stats viewing because it was OOMing the old VPS, but I think since then IIRC the VPS got more available RAM.  @leyyin Can you re-enable stats viewing?

Reply:
@hiker The comment is wrong, it says the new default is not compliant

Reply:
Unfortunately I can't because the generation scrip is badly optimized it needs more than 1 GB of ram.

Reply:
@leyyin I thought the VPS got 2GB RAM a little while back as the provider upgraded the plans?  Also is the script in a github repo so someone can work on it?  Stats are useless if we can't use them, this needs to be fixed.

Or maybe just buy a VPS with more RAM, AFAIK we're not short cash for servers.

Reply:
It got an upgrade to only 1 GB of ram.

As for the scripts, see:
- https://github.com/supertuxkart/stk-stats
- https://github.com/supertuxkart/stk-stats/issues/7

Buying more servers won't be a solution, fixing the code is.
Fix software not buy more hardware ;)